src/screens/Menu/CommunitySettings.js

```diff
@@ -0,0 +1,159 @@
+
+type Props = {
+
+};
+
+type State = {
+  email: string,
+};
+
+class CommunitySettings extends React.Component<Props, State> {
+  state = {
+    email: '',
+  }
+
+  goTo = (link: string) => () => Linking.openURL(link);
+
+  getSocialMediaItems = () => {
+    return [
+      {
+        key: 'twitter',
+        title: 'Twitter',
+        onPress: this.goTo(TWITTER_SOCIAL_ADDRESS),
+      },
+      {
+        key: 'telegram',
+        title: 'Telegram',
+        onPress: this.goTo(TELEGRAM_SOCIAL_ADDRESS),
+      },
+      {
+        key: 'youtube',
+        title: 'Youtube',
+        onPress: this.goTo(YOUTUBE_SOCIAL_ADDRESS),
+      },
+      {
+        key: 'medium',
+        title: 'Medium',
+        onPress: this.goTo(MEDIUM_SOCIAL_ADDRESS),
+      },
+      {
+        key: 'facebook',
+        title: 'Facebook',
+        onPress: this.goTo(FACEBOOK_SOCIAL_ADDRESS),
+      },
+      {
+        key: 'forum',
+        title: 'Forum',
+        onPress: this.goTo(FORUM_SOCIAL_ADDRESS),
+      },
+    ];
+  }
+
+  subscribe = () => {
+    subscribeToNewsletter(this.state.email)
+      .then(({ data }) => {
+        if (data.result === 'success') {
+          Toast.show({
+            title: 'Subscribed!',
+            type: 'success',
+            message: 'Subscribed to the newsletter successfully.',
+          });
+        } else {
+          Toast.show({
+            title: 'Subscription failed',
+            type: 'warning',
+            message: data.msg || 'Failed to subscribe to the newsletter',
+          });
+        }
+      })
+      .catch(e => {
+        Toast.show({
+          title: 'Subscription failed',
+          type: 'warning',
+          message: e.toString() || 'Failed to subscribe to the newsletter',
+        });
+      });
+  }
+
+  render() {
+    const { email } = this.state;
+
+    return (
+      <ContainerWithHeader
+        headerProps={{ centerItems: [{ title: 'Community' }] }}
+      >
+        <ScrollWrapper
+          contentContainerStyle={{ paddingHorizontal: spacing.layoutSides, paddingVertical: spacing.mediumLarge }}
+        >
+          <BaseText medium>
+            Join Pillar community, subscribe to our newsletter to receive latest news, updates and more
+          </BaseText>
+          <TextInput
+            inputProps={{
+              value: email,
+              onChange: value => this.setState({ email: value }),
+              placeholder: 'Your e-mail',
+              autoCapitalize: 'none',
+            }}
+            inputWrapperStyle={{ marginTop: spacing.mediumLarge }}
+            buttonProps={{
+              title: 'Subscribe',
+              small: true,
+              regularText: true,
+              marginRight: 12,
+              height: 32,
+              onPress: this.subscribe,
+              disabled: !isValidEmail(email),
+            }}
+            errorMessage={email && !isValidEmail(email) && 'Please enter a valid email'}
```

Comment:
Maybe we could validate and show invalid email error message on change only when it's been submitted? It will always be invalid almost till the end of typing the actual email. And user is aware of it at that time - no need to show errors :)